.github/workflows/release.yml

```
@@ -1,61 +1,25 @@
 name: Release
 
-on: 
+on:
   pull_request:
     types: [closed]
-    branches:
-    - 'release-*'
+    tags:
+      - 'v*'
 
 jobs:
   release:
-    # Only trigger if merged from pre-release-* branch
-    if: github.event.pull_request.merged == true && startsWith(github.head_ref , 'pre-release-')
+    if: github.event.pull_request.merged == true)
```

I guess this shouldn't be there
```suggestion
    if: github.event.pull_request.merged == true
```

---

Thanks! I'm not sure how to test this until we actually have a real (or fake) release PR up. It sucks that there is no easy way to evaluate if a pipeline config is correct. :)